Koha/Schema/Result/ApiKey.pm

```diff
@@ -0,0 +1,123 @@
+use utf8;
+package Koha::Schema::Result::ApiKey;
+
+# Created by DBIx::Class::Schema::Loader
+# DO NOT MODIFY THE FIRST PART OF THIS FILE
+
+=head1 NAME
+
+Koha::Schema::Result::ApiKey
+
+=cut
+
+use strict;
+use warnings;
+
+use base 'DBIx::Class::Core';
+
+=head1 TABLE: C<api_keys>
+
+=cut
+
+__PACKAGE__->table("api_keys");
+
+=head1 ACCESSORS
+
+=head2 client_id
+
+  data_type: 'varchar'
+  is_nullable: 0
+  size: 191
+
+=head2 secret
+
+  data_type: 'varchar'
+  is_nullable: 0
+  size: 191
+
+=head2 description
+
+  data_type: 'varchar'
+  is_nullable: 0
+  size: 255
+
+=head2 patron_id
+
+  data_type: 'integer'
+  is_foreign_key: 1
+  is_nullable: 0
+
+=head2 active
+
+  data_type: 'tinyint'
+  default_value: 1
+  is_nullable: 0
+
+=cut
+
+__PACKAGE__->add_columns(
+  "client_id",
+  { data_type => "varchar", is_nullable => 0, size => 191 },
+  "secret",
+  { data_type => "varchar", is_nullable => 0, size => 191 },
+  "description",
+  { data_type => "varchar", is_nullable => 0, size => 255 },
+  "patron_id",
```

Comment:
As stated above, I don't really like the idea of tieing this to a user at this level.